internal/provider/resource_cloudflare_access_group_test.go

```diff
@@ -401,6 +401,31 @@ resource "cloudflare_access_group" "%[2]s" {
 }`, accountID, rnd, githubOrg, team)
 }
 
+func testAccCloudflareAccessGroupWithIDPSAML(accountID, rnd, samlOrg, team string) string {
+	return fmt.Sprintf(`
+		resource "cloudflare_access_identity_provider" "%[2]s" {
+			account_id = "%[1]s"
+			name = "%[2]s"
+			type = "saml"
+			config {
```

Comment:
i don't think this is correct as the API request is throwing an invalid config error.

```
{
  "result": null,
  "success": false,
  "errors": [
    {
      "code": 13004,
      "message": "access.api.error.invalid_connection_config"
    }
  ],
  "messages": []
}
```